src/applications/disability-benefits/all-claims/pages/disabilityConditions/shared/utils.js

```diff
@@ -223,23 +223,15 @@ const getCurrentYear = () => new Date().getFullYear();
 export const validateApproximateDate = (errors, dateString) => {
```

Comment:
I believe the new condition date page is dependent on this function that allows any year, year and month, or full date entries as that is what Lighthouse will accept. It does not have to be a full date, but cannot be day and month or day and year.